experiment-predict/main.py

```diff
@@ -0,0 +1,19 @@
+from cortex import Cortex
+from fastapi import FastAPI
+
+app = FastAPI()
+
```

Comment:
Perhaps show at startup/init versus invoke ??

Reply:
Definitely need to add init() to load the model. Loading it in predict() should not be a best practice.